core/src/main/scala/org/apache/spark/status/api/v1/ApiRootResource.scala

```diff
@@ -164,6 +164,18 @@ private[v1] class ApiRootResource extends UIRootFromServletContext {
     }
   }
 
+  @Path("applications/{appId}/download")
+  def getEventLogs(
+    @PathParam("appId") appId: String): EventLogDownloadResource = {
+    new EventLogDownloadResource(uiRoot, appId, None)
+  }
+
+  @Path("applications/{appId}/{attemptId}/download")
```

Comment:
I am following the current API format - for example: `applications/{appId}/{attemptId}/storage/rdd/{rddId: \d+}` etc.